addons/hr_skills/static/src/scss/hr_employee.scss

```diff
@@ -1,14 +1,5 @@
-.o_hr_skills_group {
-    padding: 0 calc(var(--gutter-x) * 1)!important;
-    &:first-child {
-        padding-left: calc(var(--gutter-x) * .5)!important;
-    }
-    &:last-child {
-        padding-right: calc(var(--gutter-x) * .5)!important;
-    }
-    .o_list_renderer {
-        overflow-x: hidden!important;
-    }
+.o_hr_skills_group .o_list_renderer {
```

Comment:
As this improvement is not related to hr_org_chart design, but is still nice, could you split it in another commit ? :)